main.go

```diff
@@ -1,227 +1,200 @@
 package fixtures
 
 import (
-	"github.com/src-d/lookout-sdk/pb"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+
+	"github.com/google/go-github/github"
 )
 
 var fixtures = Fixtures{
 	{
 		Name: "new-go-file-too-long-line",
-		CommitRevision: pb.CommitRevision{
-			Base: pb.ReferencePointer{
-				InternalRepositoryURL: "https://github.com/src-d/lookout-test-fixtures",
-				ReferenceName:         "refs/heads/master",
-				Hash:                  "6a92946068897d0a6f6ffa6457f889163dcc51b5",
-			},
-			Head: pb.ReferencePointer{
-				InternalRepositoryURL: "https://github.com/src-d/lookout-test-fixtures",
-				ReferenceName:         "refs/heads/new_go_file",
-				Hash:                  "f47f238d1734881b17e6000cb9fd01248073c900",
-			},
-		},
-		PR: PullRequest{
-			URL: "https://github.com/src-d/lookout-test-fixtures/pull/14",
+		URL: PullRequestURL{
+			Owner:  "src-d",
+			Repo:   "lookout-test-fixtures",
+			Number: 14,
 		},
+		CurrentRevision: 0,
```

Comment:
I'm thinking maybe it would be nicer to have list names here instead of int.

Compare:
```
var fixtures = Fixtures{
	{
		Name: "with-many-revisions",
		URL: PullRequestURL{
			Owner:  "src-d",
			Repo:   "lookout-test-fixtures",
			Number: 14,
		},
		CurrentRevision: 4,
	}
}
```

```
var fixtures = Fixtures{
	{
		Name: "with-many-revisions",
		URL: PullRequestURL{
			Owner:  "src-d",
			Repo:   "lookout-test-fixtures",
			Number: 14,
		},
		Revisions: [
			"add file",
			"change file in multiple commits",
			"remove file",
			"introduce merge conflict",
		],
	}
}
```

it might be useful for better errors in tests.
wdyt?

Reply:
`CurrentRevison` helps to set which json snapshot of the Fixture should be loaded, see `GetRevision` and `FilenamePR`. It's not just about knowing how many revisions, but also which one is "active".

What we could do is add these revision descriptions as a complementary field, and return the current one with a new method `GetDescription()`. How does that sound to you?

Reply:
mmm... `CurrentRevison == len(Revisions)`. Does it make sense?

Reply:
What I don't like is that: 
- if you go back to revision 0 you loose information about the other revisions, and you can't go forward.
- we will end up with most of the fixtures with one revision, and the repeated string for the name and the only revision string.

Reply:
makes sense